sass/components/_view.scss

```diff
@@ -86,6 +87,50 @@
 			}
 		}
 	}
+	.cq-view-dialog
```

Comment:
maybe this should be called `.cq-view-prompt`? Dialog doesn't seem that appropriate a name to use.

Also you are missing that tiny downward pointing arrow below the overwrite button.